base_app/migrations/0024_alter_schedule_movie.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.1 on 2022-08-25 12:20
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('base_app', '0023_seat_is_occupied'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='schedule',
+            name='movie',
+            field=models.ForeignKey(default=None, on_delete=django.db.models.deletion.SET_DEFAULT, related_name='schedules', to='base_app.movie'),
+        ),
+    ]
```

Comment:
No changes are needed here but, most likely, this app should not delete data but just mark it as deleted and hide it from the clients.
Admins should be able to access that data though.